source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -382,6 +382,24 @@ def __init__(self):
             hidden=True,
             help="how long it takes to park the car, " "used especially in distributed scenario",
         )
+        parser_get.add_argument(
+            "_here_realtime_traffic",
+            type=six.text_type,
+            hidden=True,
+            help="Here, Active or not the realtime traffic information (enabled/disabled)",
+        )
+        parser_get.add_argument(
+            "_here_matrix_type",
+            type=six.text_type,
+            hidden=True,
+            help="Here, street network matrix type (simple_matrix/multi_direct_path)",
+        )
+        parser_get.add_argument(
+            "_here_max_matrix_points",
+            type=six.text_type,
```

Comment:
Same as above with type=int and controlling de upper limit value in https://github.com/CanalTP/navitia/blob/dev/source/navitiacommon/navitiacommon/default_values.py